lncfg/config.go

```diff
@@ -72,6 +72,11 @@ const (
 	// DefaultZombieSweeperInterval is the default time interval at which
 	// unfinished (zombiestate) open channel flows are purged from memory.
 	DefaultZombieSweeperInterval = 1 * time.Minute
+
+	// DefaultMaxWaitNumBlocksFundingConf is the maximum number of blocks to
+	// wait for the funding transaction to confirm before forgetting
+	// channels that aren't initiated by us. 2016 blocks is ~2 weeks.
+	DefaultMaxWaitNumBlocksFundingConf = 2016
```

Comment:
Isn't this defined twice now? I think we should just use this value everywhere now and remove the one in `funding`. Unless that will cause a circular package dependency, in which case we need to find a good place for it.